pages/battle.php

```diff
@@ -161,19 +171,30 @@ function processBattleFightEnd(BattleManager|BattleManagerV2 $battle, User $play
         if ($player->daily_tasks->hasTaskType(DailyTask::ACTIVITY_PVP)) {
             $player->daily_tasks->progressTask(DailyTask::ACTIVITY_PVP, 1);
         }
+
     } else if ($battle->isOpponentWinner()) {
         $result .= "You lose. You were taken back to your village by some allied ninja.[br]";
-        $player->health = 5;
         $player->pvp_losses++;
         $player->last_pvp_ms = System::currentTimeMs();
         $player->last_death_ms = System::currentTimeMs();
```

Comment:
Should we halve the invulnerability time for a retreat? Seems like if players are only suffering half the rep consequence they should only get half the "benefits"